src/bridge/command.rs

```diff
@@ -80,7 +80,8 @@ fn platform_exists(bin: &str) -> bool {
 }
 
 fn platform_which(bin: &str) -> Option<String> {
-    if let Some(mut which_command) = create_platform_shell_command(format!("which {}", bin)) {
+    if let Some(mut which_command) = create_platform_shell_command("which", vec![bin]) {
+        info!("Running which command: {:?}", which_command);
```

Comment:
Nit: A `info!` might be a bit too loud than actually needed. I think a `debug!` or even `trace!` might be better, the actual command executed is rarely needed imo.